### PR TITLE
Document Influx VM setup

### PR DESCRIPTION
**Pull request by yanske1**

### CODE REVIEW

File: influxdb/README.md [hunk 1]
```diff
@@ -78,3 +78,16 @@ provided such as:
...
 FILL(0)`.
+
+## GCP Setup
+
+A common InfluxDB setup will involve having InfluxDB running in a GCE VM, if
+the user is not using InfluxDB cloud. To connect ts-bridge, an App Engine
+application to a GCE VM, a
+[serverless VPC](https://cloud.google.com/vpc/docs/configure-serverless-vpc-access)
+can be configured for App Engine, and ts-bridge can then be pointed to the VM's
+internal IP address.
```
Comment (knyar) [inline influxdb/README.md:89]: I would suggest to explicitly mention that this is only necessary if InfluxDB port needs to be unreachable from public internet.

**Comment by knyar** [comment]
Did you intend to add changes to `.travis.yml` into this PR as well?

**Comment by yanske1** [comment]
Yeah, I'm suspecting mockgen is not being held to a revision is resulting in go generate ./.. changes, WIP at the moment

File: influxdb/README.md (hunk 1 above)
Comment (yanske1) [reply influxdb/README.md:89]: Good idea, added

**Comment by yanske1** [comment]
Added a TODO for now for the travis

**Pull request merged**